java/src/jmri/jmrix/loconet/LnCommandStationType.java

```diff
@@ -32,36 +32,37 @@
 @javax.annotation.concurrent.Immutable
 public enum LnCommandStationType {
 
-    //  enum value(name, canRead, progEndOp, ThrottleManager, SlotManager, supportsIdle
-    COMMAND_STATION_DCS100("DCS100 (Chief)", true, false, "LnThrottleManager", "SlotManager", true), // NOI18N
-    COMMAND_STATION_DCS240("DCS240 (Advanced Command Station)", true, false, "LnThrottleManager", "SlotManager", true), // NOI18N
-    COMMAND_STATION_DCS210("DCS210 (Evolution Command Station)", true, false, "LnThrottleManager", "SlotManager", true), // NOI18N
-    COMMAND_STATION_DCS200("DCS200", true, false, "LnThrottleManager", "SlotManager", true), // NOI18N
-    COMMAND_STATION_DCS050("DCS50 (Zephyr)", true, false, "LnThrottleManager", "SlotManager", false), // NOI18N
-    COMMAND_STATION_DCS051("DCS51 (Zephyr Xtra)", true, false, "LnThrottleManager", "SlotManager", false), // NOI18N
-    COMMAND_STATION_DB150("DB150 (Empire Builder)", false, true, "LnThrottleManager", "SlotManager", true), // NOI18N
+    //  enum value(name, canRead, progEndOp, ThrottleManager, SlotManager, supportsIdle, supports Multimeter
```

Comment:
I know this is just a comment, but for consistency, there shouldn't be a space here between supports and Multimeter.

Reply:
Thx. Done